### PR TITLE
Tw project count

### PR DESCRIPTION
**Pull request by TCW417**
Added project count value in data returned by server. Updated front end to use it instead of projects.size which is just the number of projects currently displayed.

### CODE REVIEW

File: civictechprojects/views.py [hunk 1]
```diff
@@ -206,6 +206,8 @@ def projects_list(request):
         else:
             project_list = projects_by_sortField(project_list, '-project_date_modified')
 
+        project_count = len(project_list)
+        print('project_count', project_count)
```
Comment (marlonkeating) [inline civictechprojects/views.py]: Remove count logging

File: civictechprojects/views.py [hunk 2]
```diff
@@ -272,11 +274,13 @@ def projects_by_roles(tags):
     return Project.objects.filter(positions__in=positions)
 
 
-def projects_with_meta_data(projects, project_pages):
+def projects_with_meta_data(projects, project_pages, project_count):
+    print('numProjects', project_count)
```
Comment (marlonkeating) [inline civictechprojects/views.py]: Remove logging

File: democracylab_environment_variables.sh [hunk 3]
```diff
@@ -37,7 +37,7 @@ export POSITION_DESCRIPTION_EXAMPLE_URL='https://docs.google.com/document/d/142N
 export VOLUNTEER_CONCLUDE_SURVEY_URL='https://docs.google.com/forms/d/e/1FAIpQLSd4e9FQYX3ARPg7qz1ct5qM_bQW_kkEvQaMuM5LV9Ma1NDgbA/viewform'
 
 # Configure number of projects to display per server request
-export PROJECTS_PER_PAGE=20
+export PROJECTS_PER_PAGE=2
```
Comment (marlonkeating) [inline democracylab_environment_variables.sh]: This isn't a good default unless you're specifically testing pagination, so best to keep it at 20 I think.

Comment (TCW417) [reply democracylab_environment_variables.sh]: Oops

**Pull request merged**